src/main/resources/db/migration/V1_90__elinks_alter_judicial_user_profile.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE dbjudicialdata.judicial_user_profile ALTER COLUMN title TYPE VARCHAR(256);
+
+ALTER TABLE dbjudicialdata.judicial_user_profile ALTER COLUMN initials TYPE VARCHAR(256);
+
+ALTER TABLE dbjudicialdata.judicial_user_profile ALTER COLUMN known_as TYPE VARCHAR(256);
+
+COMMIT;
```

Comment:
@arshinsalim request please add script in functional test as well to make sure in sync :)